.idea/runConfigurations/_template__of_py_test.xml

```diff
@@ -50,4 +49,4 @@
     <option name="_new_targetType" value="&quot;PYTHON&quot;" />
     <method v="2" />
   </configuration>
-</component>
+</component>
```

Comment:
It looks like you've introduced a formatting issue here (all files should end with newline characters). Please revert this change